NTManager.py

```diff
@@ -1,10 +1,23 @@
 from networktables import NetworkTables
+import main
 
 
 class NTManager:
-    nt = None
 
     def __init__(self, table_key, robot_ip='10.59.90.2'):
         NetworkTables.initialize(robot_ip)
         self.nt = NetworkTables.getTable(table_key)
+        self.nt.addEntryListener(main.nt_settings_listener)
+
```

Comment:
This creates a "recursive" dependency. (NTManager imports main and main imports NTManager). 
pass the nt_settings_listener as a  callback function in __init__.